src/ray/core_worker/common.h

```diff
@@ -118,7 +119,14 @@ struct ActorCreationOptions {
         concurrency_groups(concurrency_groups.begin(), concurrency_groups.end()),
         execute_out_of_order(execute_out_of_order),
         max_pending_calls(max_pending_calls),
-        scheduling_strategy(scheduling_strategy){};
+        scheduling_strategy(scheduling_strategy) {
+    // Check that resources is a subset of placement resources.
+    for (auto &resource : resources) {
```

Comment:
Can you add the same check inside task_options?

Reply:
same as above, I think it's okay to leave it as is for task options (since the two are identical) and just remove the variable in the caller